apis/openstack-cinder/src/main/java/org/jclouds/openstack/cinder/v2/domain/AbsoluteLimit.java

```diff
@@ -0,0 +1,239 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.openstack.cinder.v2.domain;
+
+import com.google.common.base.Objects;
+
+import javax.inject.Named;
+import java.beans.ConstructorProperties;
+
+public class AbsoluteLimit {
+
+    @Named("totalSnapshotsUsed")
+    private final int totalSnapshotsUsed;
+    @Named("maxTotalBackups")
+    private final int maxTotalBackups;
+    @Named("maxTotalVolumeGigabytes")
+    private final int maxTotalVolumeGigabytes;
+    @Named("maxTotalSnapshots")
+    private final int maxTotalSnapshots;
+    @Named("maxTotalBackupGigabytes")
+    private final int maxTotalBackupGigabytes;
+    @Named("totalBackupGigabytesUsed")
+    private final int totalBackupGigabytesUsed;
+    @Named("maxTotalVolumes")
+    private final int maxTotalVolumes;
+    @Named("totalVolumesUsed")
+    private final int totalVolumesUsed;
+    @Named("totalBackupsUsed")
+    private final int totalBackupsUsed;
+    @Named("totalGigabytesUsed")
+    private final int totalGigabytesUsed;
+
+    @ConstructorProperties({
+            "totalSnapshotsUsed", "maxTotalBackups", "maxTotalVolumeGigabytes", "maxTotalSnapshots", "maxTotalBackupGigabytes",
```

Comment:
this should be similar to the fields the return in the api. does the field are totalSnapshotsUsed or total_snapshots_used (for example) ?